mathematical_puzzles/Q1/Main.scala

```diff
@@ -0,0 +1,20 @@
+import scala.util.control.Breaks
+
+object Main extends App{
+  var i:Int = 10
+
+  while(true){
+    if(isPalindromes(Integer.toString(i)) && isPalindromes(Integer.toOctalString(i)) && isPalindromes(Integer.toBinaryString(i))){
+      println(i)
+      val b = new Breaks
+      b.break()
+    }else{
+      i+=1
+    }
+  }
```

Comment:
Thank you!  I will never use `scala.util.control.Breaks` and changed to Iterator https://github.com/okoysm/study-scala/commit/279333e55073e12f3844971da515a2e0d1b20d2f .